src/frontend/sider/Fagsak/Behandling/Sider/Vilkårsvurdering/Varsel/ManglendeFinnmarkmerkingVarsel.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { Alert, BodyLong, Heading, Table, VStack } from '@navikt/ds-react';
+
+import { isoDatoPeriodeTilFormatertString } from '../../../../../../utils/dato';
+import { useBehandlingContext } from '../../../context/BehandlingContext';
+
+export const ManglendeFinnmarkmerkingVarsel: React.FC = () => {
```

Comment:
Er vel ikke så viktig med `React.FC` her

```suggestion
export const ManglendeFinnmarkmerkingVarsel = () => {
```